documentation/modules/security/proc-renewing-your-own-ca-certificates.adoc

```diff
@@ -16,92 +17,68 @@ The procedure describes the renewal of CA certificates in PEM format.
+_CA-CERTIFICATE-SECRET_ is the name of the `Secret`, which is `_KAFKA-CLUSTER-NAME_-cluster-ca-cert` for the cluster CA certificate and `_KAFKA-CLUSTER-NAME_-clients-ca-cert` for the clients CA certificate.
 +
-For example `ca-2018-09-27T17-32-00Z.crt`.
+Following an example of the `Secret` related to the cluster CA certificate for a Kafka cluster named `my-cluster`.
 +
-[source,shell,subs="+quotes"]
-mv ca.crt ca-$(date -u -d"$(openssl x509 -enddate -noout -in ca.crt | sed 's/.*=//')" +'%Y-%m-%dT%H-%M-%SZ').crt
+[source,yaml,subs=attributes+]
+----
+apiVersion: v1
+kind: Secret
+data:
+  ca.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0F... <1>
+metadata:
+  annotations:
+    strimzi.io/ca-cert-generation: "0" <2>
+  labels:
+    strimzi.io/cluster: my-cluster
+    strimzi.io/kind: Kafka
+  name: my-cluster-cluster-ca-cert
+  #...
+type: Opaque
+----
+<1> Current CA certificate base64 encoded
+<2> Current CA certificate generation
 
-. Copy your new CA certificate into the directory, naming it `ca.crt`:
+.. Encode your new CA certificate into base64.
 +
 [source,shell,subs="+quotes"]
-cp _PATH-TO-NEW-CERTIFICATE_ ca.crt
+cat _PATH-TO-NEW-CERTIFICATE_ | base64
 
-. Put your CA certificate in the corresponding `Secret`.
+.. Update the CA certificate.
 +
-.. Delete the existing secret:
+Copy the new CA certificate base64 encoded got from the previous step into the `ca.crt` field of the `data` section in the `Secret`.
 +
-[source,shell,subs="+quotes"]
-kubectl delete secret _CA-CERTIFICATE-SECRET_
+.. Increase the CA certificate generation.
 +
-_CA-CERTIFICATE-SECRET_ is the name of the `Secret`, as returned in the first step.
+Updated the `strimzi.io/ca-cert-generation` annotation with a higher incremental value.
+For example, change `strimzi.io/ca-cert-generation=0` to `strimzi.io/ca-cert-generation=1`.
 +
-Ignore any "Not Exists" errors.
 
-.. Recreate the secret:
+. Save changes and exit.
 +
-[source,shell,subs="+quotes"]
-kubectl create secret generic _CA-CERTIFICATE-SECRET_ --from-file=.
-
-. Delete the directory you created:
-+
-[source,shell,subs="+quotes"]
-----
-cd ..
-rm -r new-ca-cert-secret
-----
+Save the changes to the `Secret` and exit.
 
-. Put your CA key in the corresponding `Secret`.
-
-.. Delete the existing secret:
-+
-[source,shell,subs="+quotes"]
-kubectl delete secret _CA-KEY-SECRET_
-+
-_CA-KEY-SECRET_ is the name of CA key, which is `_KAFKA-CLUSTER-NAME_-cluster-ca` for the cluster CA key and `_KAFKA-CLUSTER-NAME_-clients-ca` for the clients CA key.
+When Strimzi generates certificates, the certificate generation annotation is automatically incremented by the Cluster Operator.
+For manual renewal of your own CA certificates, set the annotations with a higher incremental value.
+The annotations need a value higher than the ones from the current Secret so the Cluster Operator can roll the pods and update the certificates.
+The `strimzi.io/ca-cert-generation` has to be incremented on each CA certificate renewal.
```

Comment:
Is this still needed? wasn't that covered above?

Reply:
This was just a kind of summary about generation and how it differs from the auto-renewal. If you think it sounds like a step to do and could be misleading, I can remove it. Wdyt?